src/module-elasticsuite-core/Search/Request/SortOrder/SortOrderBuilder.php

```diff
@@ -165,7 +165,12 @@ private function addDefaultSortOrders($orders, MappingInterface $mapping)
     private function getSortOrderParams(FieldInterface $field, array $sortOrderParams)
     {
         $sortOrderParams['field']   = $field->getMappingProperty(FieldInterface::ANALYZER_SORTABLE);
-        $sortOrderParams['missing'] = $field->getSortMissing($sortOrderParams['direction']);
+
+        // @codingStandardsIgnoreStart
+        if (!in_array($sortOrderParams['missing'] ?? false, [SortOrderInterface::MISSING_FIRST, SortOrderInterface::MISSING_LAST]))
```

Comment:
You forgot the opening curly bracket of the if :grimacing:

Reply:
> You forgot the opening curly bracket of the if 😬 (well I did, but you did too !)

Sh**, sorry :confounded: 

> (Feel free to squash your commits after the fix)

Done :heavy_check_mark: